apps/frontpage/app/docs/sitemap.ts

```diff
@@ -0,0 +1,40 @@
+import { MetadataRoute } from 'next';
+import { generateDocsTree } from '../../lib/get-tree';
+import { TreeProps, docsVersions } from '@repo/utils';
+
+export default function sitemap(): MetadataRoute.Sitemap {
+  const latestVersion = docsVersions[0];
+  const tree = generateDocsTree(`content/docs/${latestVersion.id}`);
+
+  const flattenedTree: TreeProps[] = [];
+
+  const flattenTree = (tree: TreeProps[]) => {
+    tree.forEach((node) => {
+      if (node.children) {
+        flattenedTree.push(node);
+        flattenTree(node.children);
+      }
+      flattenedTree.push(node);
+    });
+  };
+
+  flattenTree(tree);
+
+  const filteredTree = flattenedTree
+    .filter((node) => node.type !== 'directory')
+    .filter((node) => node.draft !== true);
+
+  const docs = filteredTree.map((node) => ({
+    url: `https://storybook.js.org${node.slug}`.replace(
+      `${latestVersion.id}/`,
+      '',
+    ),
```

Comment:
Not blocking

TODO: Refactor this to ensure it only ever replaces versions at the start of the route. If a page slug ever contains a version id, e.g. `migrate-from-8.0`, this will break.

Reply:
The real refactor that need to happen is to make sure the slug reflect the real scenario so we don't have to do any modifications here.

Reply:
This should not be much better.